osmosis/README.md

```diff
@@ -75,20 +75,18 @@ const useCounterStore = () => {
   };
 };
 
-let [CounterContext, wrapCounter, counterRef] = setupStore(useCounterStore);
+let CounterStore = setupStore(useCounterStore);
 
-export { CounterContext, wrapCounter };
-
-export default counterRef;
+export default CounterStore;
 ```
 
 ```jsx
 //counter.js
 import React, { useContext } from 'react';
-import { CounterContext } from './counter.store';
+import { CounterStore } from './counter.store';
 
 export default () => {
-  const [counterContext] = useContext(CounterContext);
+  const [counterContext] = useContext(CounterStore.Context);
```

Comment:
can we update this example while we're here to call the object returned the store and not the context?

`const [counterStore] = useContext(CounterStore.Context);`